terraform/rtmp-server/bootstrap.sh

```diff
@@ -41,7 +54,6 @@ tar -C /usr/local -xzf /tmp/golang/go1.11.2.linux-amd64.tar.gz
 . /etc/profile
 
 # Create directory for generating client keys
-mkdir /root/client-keys
```

Comment:
If you delete a line of code, please make sure to address the associated comments and blank lines